Workbooks/UsageMetrics/Usage - Users/Usage - Users.workbook

```diff
@@ -0,0 +1,710 @@
+            "id": "37b05d6c-ab7c-4190-94d8-4973fcf06c49",
+            "version": "KqlParameterItem/1.0",
+            "name": "TimeRange",
+            "label": "During",
+            "type": 2,
+            "isRequired": true,
+            "query": "datatable(value:string, text:string)[\r\n'1h', 'Last hour', \r\n'24h', 'Last 24 hours', \r\n'48h', 'Last 48 hours', \r\n'3d', 'Last 3 days',\r\n'7d', 'Last 7 days',\r\n'14d', 'Last 14 days',\r\n'30d', 'Last 30 days',\r\n'60d', 'Last 60 days',\r\n'90d', 'Last 90 days'\r\n]",
+            "value": "24h",
+            "typeSettings": {
+              "additionalResourceOptions": []
+            },
+            "queryType": 0,
+            "resourceType": "microsoft.insights/components"
+          },
+          {
+            "id": "3774b890-2a45-46d8-9a92-01145cc1c4ba",
+            "version": "KqlParameterItem/1.0",
+            "name": "WithGrain",
+            "type": 1,
+            "isRequired": true,
+            "query": "let grainList = datatable(grain:string, text:string)[\r\n'3m', '1h', \r\n'1h', '24h',\r\n'2h', '48h',\r\n'3h', '3d',\r\n'1d','7d',\r\n'1d','14d',\r\n'1d','30d',\r\n'1d','60d',\r\n'1d','90d'\r\n];\r\ngrainList\r\n| where text == '{TimeRange}'\r\n| project grain",
+            "isHiddenWhenLocked": true,
+            "queryType": 0,
+            "resourceType": "microsoft.insights/components"
+          },
+          {
+            "id": "fd41f402-40ca-4dc5-bd29-38ec93f9d521",
+            "version": "KqlParameterItem/1.0",
+            "name": "UserEvent",
+            "label": "Selected events",
+            "type": 2,
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "query": "let timeRange = {TimeRange};\r\n   union {Whoused}\r\n    | where timestamp > ago(timeRange)\r\n    | project name, itemType\r\n    | order by itemType,name asc\r\n    | summarize test = count() by itemType,name\r\n    | project Id=name, Title=name, Selected=false\r\n    | union (\r\ndatatable(Id:string, Title:string, Selected:boolean)[\r\n'*', 'All', true\r\n])\r\n\r\n",
+            "typeSettings": {
+              "additionalResourceOptions": [],
+              "showDefault": false
+            },
+            "queryType": 0,
+            "resourceType": "microsoft.insights/components"
+          },
+          {
+            "id": "7a61f92e-cd11-494a-8790-b2e32f591a11",
+            "version": "KqlParameterItem/1.0",
+            "name": "ByType",
+            "label": "By type",
+            "type": 2,
+            "isRequired": true,
+            "value": "Time",
+            "typeSettings": {
+              "additionalResourceOptions": []
+            },
+            "jsonData": "[\"Time\", \"Property\"]"
+          },
+          {
+            "id": "8cd9003c-8e76-4d21-8b22-b7a11197f42f",
+            "version": "KqlParameterItem/1.0",
+            "name": "CountBy",
+            "label": "By value",
```

Comment:
you should also be setting the description/explanation on all these parameters so they have infoballoons in the workbook as well?